fondant/cli.py

```diff
@@ -0,0 +1,96 @@
+import argparse
+import importlib.util
+import re
+
+from fondant.compiler import DockerCompiler
+from fondant.pipeline import Pipeline
+
+
+class PipelinePythonReference(argparse.Action):
```

Comment:
I did not find anything standard on importstrings (`file.py:my_object`) so I made my own parser.

Reply:
Any reason to go for a custom `Action` instead of a custom `type`? I think a custom `type` could be a bit simpler.

FYI, [reference implementation](https://github.com/encode/uvicorn/blob/master/uvicorn/importer.py#L9) from `uvicorn`.